src/routes/PetPage/PetPage.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Section } from '../../components/Utils/Utils';
+import PetsService from '../../services/petsService';
+import { Link } from 'react-router-dom';
+
+export default function PetPage () {
+  const params = useParams();
+  const [pet, setPet] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (!pet) {
+      PetsService.getPet(params.petID)
+        .then(pet => setPet(pet))
+        .catch(e => {
+          setError(e.error);
+        });
+    }
+  });
+
+  const renderFavoriteIcon = () => {
+    // TODO
+    if (true) {
+      return <span className='text-danger'><FontAwesomeIcon icon='heart' /></span>;
+    }
+    return <span className='text-danger'><FontAwesomeIcon icon={['far', 'heart']} /></span>;
+  }
+
+  const renderPetBio = () => {
+    return (
+      <div className='col-md m-1'>
+        <h2>
+          Hi, I'm {pet.Name}!
+          <span> </span>
+          {renderFavoriteIcon()}
+        </h2>
+        <h6>{pet.TypeOfAnimal}, {pet.Breed}</h6>
+        <table className='table table-sm table-borderless'>
+          <tbody>
+            <tr>
+              <td scope='row'>Sex:</td>
+              <td>{pet.Sex}</td>
+            </tr>
+            <tr>
+              <td scope='row'>Age:</td>
+              <td>{pet.Age} {pet.Age === 1 ? 'yr' : 'yrs'}</td>
```

Comment:
This is a total nitpick, but I think the shorthand 'yr' looks kind of out of place when everything else on the page is spelled out. Thoughts on using "Year/Years old" instead? Everything else looks great!